services_and_applications/slate/overview.rst

```diff
@@ -0,0 +1,37 @@
+*****************************
+Overview
+*****************************
+
+*Slate* is the high-level service name referring to two user
+facing OpenShift clusters.
+
+| OpenShift is a container orchestration system built on top of Kubernetes and supported by Red Hat. 
+  Kubernetes provides a rich API and workload primitives that allows users to manage the application 
+  deployments of long running services such as web servers and databases.
+
+| OpenShift is a Kubernetes Certified Platform. This means that all of the functionality that is
+  in Kubernetes is also implemented in OpenShift. OpenShift has extra functionality built on 
+  top though, such as the container builder and integrated registry services.
```

Comment:
I say we move this to under the "What is OpenShift" header at the bottom

Reply:
sound good. will do. Thanks!